R/ml_feature_transformation.R

```diff
@@ -391,3 +391,30 @@ ft_regex_tokenizer <- function(x,
 #     setBinary      = ensure_scalar_boolean(binary)
 #   ))
 # }
+
+#' Feature Tranformation -- CountVectorizer
+#'
+#' Extracts a vocabulary from document collections.
+#'
+#' @template roxlate-ml-transformation
+#'
+#' @export
+ft_count_vectorizer <- function(x,
+                                input.col = NULL,
+                                output.col = NULL,
+                                min_df = NULL,
```

Comment:
Yes for sure; however, it does make me want to consider renaming them all to `_` at some point since we've named all our public parameters with `_`.